crates/rocksdb-store/src/lib.rs

```diff
@@ -60,8 +60,7 @@ use bridge::schemas::{
 };
 pub const PROVER_COLUMN_FAMILIES: &[ColumnFamilyName] = &[
     SequenceSchema::COLUMN_FAMILY_NAME,
-    prover::schemas::ProverTaskIdSchema::COLUMN_FAMILY_NAME,
-    prover::schemas::ProverTaskSchema::COLUMN_FAMILY_NAME,
```

Comment:
Why do we not need the prover tasks though?

Reply:
Prover Tasks will be implied by the use of `ProofId`.
For example `ProofId::L1Batch(u64, u64)` will automatically imply that it should get proofs from`ProofId::BtcBlockspace(u64)` till `ProofId::BtcBlockspace(u64)`.

Reply:
That's cool